bin/heatmapper

```diff
@@ -511,8 +555,8 @@ def main(args):
     if args.outFileNameMatrix:
         hm.saveMatrixValues(args.outFileNameMatrix)
 
-    if args.outFileNameData:
-        hm.saveTabulatedValues(args.outFileNameData)
+    #if args.outFileNameData:
```

Comment:
The function was not really working, thus I commented it out long time ago. Furthermore, for the users that want the raw data, they can always unzip the matrix that is passed from computeMatrix to heatmapper.